src/arcsf/eval/scripts/analysis.py

```diff
@@ -0,0 +1,100 @@
+import json
+import os
+
+import matplotlib.pyplot as plt
+import yaml
+
+from arcsf.eval.utils import get_analysis_values, get_metrics, plot_cdf, plot_scatter
+
+
+def run_analysis(
+    base_model_path: str,
+    test_model_path: str,
+    verbose: bool = False,
+    plotting: bool = False,
+) -> dict[str, float, float, float, float]:
+    """
```

Comment:
Type hinting for dicts is just key type then value type (not one for each value)


```suggestion
) -> dict[str, float]:
    """
```